extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/KubernetesCommonHelper.java

```diff
@@ -139,6 +143,7 @@ public static Optional<Project> createProject(ApplicationInfoBuildItem app,
                             buildInfo, project.getScmInfo()));
 
         } catch (Exception e) {
+            LOG.debugv(e, "Couldn't create project for {0} application", name);
```

Comment:
Should it be debug or does it have value for the user?

Reply:
I debated using info level… the problem is that the exception you usually get is not really actionable as-is since you get exceptions like the one seen in #36041. Maybe if dekorate was returning more informative exceptions, it would make more sense to use an info level here.